data/sql_setup/prepare_schedule_e.sql

```diff
@@ -217,7 +217,7 @@ end
 $$ language plpgsql;
 
 drop trigger if exists ofec_sched_e_queue_trigger on fec_vsum_sched_e;
-create trigger ofec_sched_e_queue_trigger before insert or update or delete
+create trigger ofec_sched_e_queue_trigger instead of insert or update or delete
```

Comment:
Maybe I am misunderstanding, but because it is going off of a view instead of a table, it needs the "instead of" rather than before.  I thought it should be fulfilling the same function that it did before. I remember reading up on it, but now I can't find where it came from, so I could be wrong.